tensorboard/webapp/plugins/npmi/views/annotations_list/header/header_component.scss

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+
+:host {
+  border-bottom: 2px solid mat-color($tb-foreground, border);
+  display: flex;
+  height: 28px;
+  align-items: flex-end;
```

Comment:
`flex-end` aligns this pretty close to the bottom of the box. Which is maybe what you want but based on your previous iteration it seemed like `baseline` was more appropriate.